Engines/Wine/QuickScript/Custom Installer Script/script.js

```diff
@@ -1,14 +1,12 @@
 include("engines.wine.quick_script.installer_script");
 
-function CustomInstallerScript() {
-    InstallerScript.call(this);
+class CustomInstallerScript extends InstallerScript {
+    constructor() {
+        super();
+    }
+
+    installationCommand(installationCommand) {
```

Comment:
I don't like the names of our `QuickScript` methods. They are named like accessor methods and fields but function as fluid API methods, which return the object itself. To make this more clear and to clear naming conflicts between the method name and the underlying field I propose to rename the methods to:

```javascript
withInstallationCommand(installationCommand) {
   this.installationCommand = installationCommand;
   return this;
}
```

Again this would also be more in line with our Java implementation.

Reply:
Agreed

Reply:
Can you open a new issue to track this?

Reply:
See @1021